bin/impl/fluo_deploy/util.py

```diff
@@ -108,27 +109,49 @@ def setup_boto(lib_dir):
     print "Finished downloading Boto"
   sys.path.insert(0, boto_lib_dir)
 
-def parse_args(input=None):
+def parse_args(hosts_dir, input=None):
   parser = OptionParser(
-            usage="fluo-deploy [options] <action> <cluster_name>"
+            usage="fluo-deploy [options] <action>"
             + "\n\nwhere <action> can be:\n  launch - Launch cluster in EC2\n  status - Check status of EC2 cluster\n"
-            + "  setup - Setup Fluo and its dependencies on cluster\n  ssh - SSH to cluster leader node\n"
-            + "  kill - Kill Fluo and its dependencies on cluster\n  terminate - Terminate EC2 cluster",
+            + "  setup - Setup Fluo and its dependencies on cluster\n  config - Print configuration for that cluster\n"
```

Comment:
This help could mention `all`.  Why not put each action's help on a separate line in source code?